.gitignore

```diff
@@ -1,4 +1,25 @@
+# Binary files
+build/
+builddir/
```

Comment:
This is not needed (at least with modern versions of Meson):

https://mesonbuild.com/FAQ.html#how-do-i-ignore-the-build-directory-in-my-vcs

Only possible by enforcing out of tree builds. :D

Reply:
Good point, I'll remove these.